added detail about throttled writes (#4081)
influxdata/docs-v2: content/enterprise_influxdb/v1.9/administration/configure/config-data-nodes.md

By default, this option is set to false and writes are processed in the order that they are received. This means if any points are in the hinted handoff (HH) queue for a shard, all incoming points must go into the HH queue.

If set to `true`, writes may process in a different order than they were received. This can reduce the time required to drain the HH queue and increase throughput during recovery.

> Whn a remote write fails, the database will send data to the hinted handoff (HH) queue for a short period of time before allowing incoming remote writes to be retried again. This time period grows with each failed write. The greater the number of successive write failures, the longer a node will delay writes before retrying them. Consecutive write failures will cause increased delay times before retrying writes in both emptying HH queues and in remote writes.

Reviewer: "... before retrying them, up to a maximum set by the configuration parameter `retry-max-interval`.  Consecutive..."

Author: @davidby-influx I'll set up a quick meeting if needed - are you saying that this is not the place to discuss  throttled writes?  Also, if there is another parameter that needs to be configured as part of this, then it should be addressed as part of this issue.

Author: @davidby-influx can you review changes made and let me know if we need a quick Zoom to add any updates?  Thank you!

> Whn a remote write fails, the database will send data to the hinted handoff (HH) queue for a short period of time before allowing incoming remote writes to be retried again. This time period grows with each failed write. The greater the number of successive write failures, the longer a node will delay writes before retrying them. Consecutive write failures will cause increased delay times before retrying writes in both emptying HH queues and in remote writes.

Reviewer 2: ```suggestion
When a remote write fails, the database sends data to the hinted handoff (HH) queue for a short period of time (up to a maximum time set in the configuration parameter [retry-max-interval](/enterprise_influxdb/v1.9/administration/configure/config-data-nodes/#retry-max-interval)) before retrying the failed writes. Each successive write failure extends the delay in retrying failed writes in the HH queue and retrying remote writes.
```

> This could result in temporarily large HH queues and more partial write errors if the [write consistency](https://docs.influxdata.com/enterprise_influxdb/v1.9/concepts/clustering/#write-consistency) level is not set to `any` when hinted handoff is written, but fewer "remote write failed" errors as the HH queues empty, and much lower memory usage when a node is unavailable. This is because remote writes, which keep data in memory while timing out, will be suppressed. Note that only **repeated** write failures will slow down the retry rate.

Reviewer 2: "...if, but, and..." clause may be difficult to digest

Can we rephrase to clarify? Maybe helpful to re-order clauses and shorten sentences to clarify?

With successive write failures, if the [write consistency](https://docs.influxdata.com/enterprise_influxdb/v1.9/concepts/clustering/#write-consistency) level is not set to `any` when hinted handoff is written, you may see the following:

- temporarily large HH queues
- more partial write errors
- fewer "remote write failed" errors as the HH queues empty (This is because remote writes, which keep data in memory while timing out, will be suppressed.?)
- much lower memory usage when a node is unavailable 

Note that only **repeated** write failures will slow down the retry rate.

> This could result in temporarily large HH queues and more partial write errors if the [write consistency](https://docs.influxdata.com/enterprise_influxdb/v1.9/concepts/clustering/#write-consistency) level is not set to `any` when hinted handoff is written, but fewer "remote write failed" errors as the HH queues empty, and much lower memory usage when a node is unavailable. This is because remote writes, which keep data in memory while timing out, will be suppressed. Note that only **repeated** write failures will slow down the retry rate.

Reviewer 2: "...if, but, and..." clause may be difficult to digest

Can we rephrase to clarify? Maybe helpful to re-order clauses and shorten sentences to clarify?

With successive write failures, if the [write consistency](https://docs.influxdata.com/enterprise_influxdb/v1.9/concepts/clustering/#write-consistency) level is not set to `any` when hinted handoff is written, you may see the following:

- temporarily large HH queues
- more partial write errors
- fewer "remote write failed" errors as the HH queues empty (This is because remote writes, which keep data in memory while timing out, will be suppressed.?)
- much lower memory usage when a node is unavailable

Note that only repeated write failures will slow down the retry rate.

Author: re-worded

> **Do not enable if your use case involves updating points, which may cause points to be overwritten.** To overwrite an existing point, the measurement name, tag keys and values (if the point includes tags), field keys, and timestamp all have to be the same as a previous write. 

Reviewer 2: ```suggestion
**Do not enable if your use case involves updating points because points may be overwritten.**  Overwrites happen when a point's measurement name, tag keys and values (if the point includes tags), field keys, and timestamp are the same as a previous write. 
```

Author: fixed

By default, this option is set to false and writes are processed in the order that they are received. This means if any points are in the hinted handoff (HH) queue for a shard, all incoming points must go into the HH queue.

If set to `true`, writes may process in a different order than they were received. This can reduce the time required to drain the HH queue and increase throughput during recovery.

With successive write failures, if the [write consistency](https://docs.influxdata.com/enterprise_influxdb/v1.9/concepts/clustering/#write-consistency) level is not set to `any` when hinted handoff is written, you may see the following:

- temporarily large HH queues
- more partial write errors
- fewer "remote write failed" errors as the HH queues empty (This is because remote writes, which keep data in memory while timing out, will be suppressed.)

> - much lower memory usage when a node is unavailable

Author: re-wrote based on feedback
